react-native-gutenberg-bridge/ios/Gutenberg.swift

```diff
@@ -104,8 +108,9 @@ extension Gutenberg: RCTBridgeDelegate {
     public func extraModules(for bridge: RCTBridge!) -> [RCTBridgeModule]! {
         let aztecManager = RCTAztecViewManager()
         aztecManager.attachmentDelegate = dataSource.aztecAttachmentDelegate()
-
-        return [bridgeModule, aztecManager]
+        var modules:[RCTBridgeModule] = [bridgeModule, aztecManager]
+        modules.append(contentsOf: extraModules)
+        return modules
```

Comment:
I think you can just do `return [bridgeModule, aztecManager] + extraModules` here